$NetBSD: patch-aa,v 1.6 2012/12/02 12:33:24 wen Exp $
Add the python version to the name of script and man file
--- setup.py.orig	2012-12-02 19:33:28.000000000 +0000
+++ setup.py
@@ -274,10 +274,10 @@ setup(
       keywords = "Math CAS",
       url = 'http://code.google.com/p/sympy',
       packages = ['sympy'] + modules + tests,
-      scripts = ['bin/isympy'],
+      scripts = ['bin/isympy' + sys.version[0:3]],
       ext_modules = [],
       package_data = { 'sympy.utilities.mathml' : ['data/*.xsl'] },
-      data_files = [('share/man/man1', ['doc/man/isympy.1'])],
+      data_files = [('man/man1', ['doc/man/isympy' + sys.version[0:3] + '.1'])],
       cmdclass    = {'test': test_sympy,
                      'bench': run_benchmarks,
                      'clean': clean,
